manifests/n/Nlitesoft/NTLite/2023.9.9419/Nlitesoft.NTLite.installer.yaml

```
@@ -0,0 +1,23 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Nlitesoft.NTLite
+PackageVersion: 2023.9.9419
+Platform:
+- Windows.Desktop
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
+  InstallerUrl: https://downloads.ntlite.com/files/NTLite_setup_x64.exe
+  InstallerSha256: 6892F30F6D300C448F19A713798562187AABC673876B86C9B800012C9B0DF50F
+- Architecture: x64

```

```suggestion
- Architecture: x64
  InstallerUrl: https://downloads.ntlite.com/files/NTLite_setup_x64.exe
  InstallerSha256: 6892F30F6D300C448F19A713798562187AABC673876B86C9B800012C9B0DF50F
- Architecture: x86
```
Hi @auanasgheps,

The architectures got swapped.